api/migrations/versions/cbf81e4d5757_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
```

Comment:
This file needs an updated `down_revision`

Reply:
Fixed.